main.go

```diff
@@ -243,6 +243,10 @@ func setupControllers(mgr ctrl.Manager, sw *watch.ControllerSwitch, tracker *rea
 	}
 
 	mutationCache := mutation.NewSystem()
+	err = mutationCache.WarmCache(mgr)
```

Comment:
I think we can integrate with Oren's readiness tracker code. Doing so would make it so the webhook isn't accessible until all resources are loaded by the operator.

Reply:
Using readiness tracker now